jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/audit/CaseLogCleanupCommand.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.casemgmt.impl.audit;
+
+import java.util.Date;
+
+import javax.persistence.EntityManagerFactory;
+
+import org.jbpm.process.audit.JPAAuditLogService;
+import org.jbpm.process.core.timer.DateTimeUtils;
+import org.jbpm.runtime.manager.impl.jpa.EntityManagerFactoryManager;
+import org.kie.api.executor.Command;
+import org.kie.api.executor.CommandContext;
+import org.kie.api.executor.ExecutionResults;
+import org.kie.api.executor.Reoccurring;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Log clean up command that aims at doing house keeping of audit/log tables used in jBPM:
+ * <ul>
+ * 	<li>process related audit logs (process instance, node instance, variables)</li>
+ * 	<li>task related audit logs (audit task, task events)</li>
+ * 	<li>executor related data (requests and errors)</li>
+ * </ul>
+ * Command by default is auto configured to run once a day from the time it was initially scheduled though it can be reconfigured
+ * in terms of frequency when it is executed and if it shall run multiple times at all.<br/>
+ * Following is a complete list of accepted parameters:
+ * <ul>
+ * 	<li>SkipProcessLog - indicates if clean up of process logs should be omitted (true|false)</li>
+ * 	<li>SkipTaskLog - indicates if clean up of task logs should be omitted (true|false)</li>
+ * 	<li>SkipExecutorLog - indicates if clean up of executor logs should be omitted (true|false)</li>
+ * 	<li>DateFormat - date format for further date related params - if not given yyyy-MM-dd is used (pattern of SimpleDateFormat class)</li>
+ * 	<li>EmfName - name of entity manager factory to be used for queries (valid persistence unit name)</li>
+ * 	<li>SingleRun - indicates if execution should be single run only (true|false)</li>
+ * 	<li>NextRun - provides next execution time (valid time expression e.g. 1d, 5h, etc)</li>
+ * 	<li>OlderThan - indicates what logs should be deleted - older than given date</li>
+ * 	<li>OlderThanPeriod - indicated what logs should be deleted older than given time expression (valid time expression e.g. 1d, 5h, etc)</li>
+ * 	<li>ForProcess - indicates logs to be deleted only for given process definition</li>
+ * 	<li>ForDeployment - indicates logs to be deleted that are from given deployment id</li>
```

Comment:
I don't see usage of all of these parameters in the actual execution of the command.